Common/DTO/Candle.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace DTO
+{
+    public class Candle
```

Comment:
I'm not sure if it should still be called candle as chart is no longer a candle chat, but ok